src/scene_battle_rpg2k.cpp

```diff
@@ -362,6 +362,10 @@ bool Scene_Battle_Rpg2k::ProcessBattleAction(Game_BattleAlgorithm::AlgorithmBase
 		}
 	}
 
+	if (Input::IsPressed(Input::CANCEL)) {
+		return false;
+	}
```

Comment:
> "4.11: RM2000: If you keep CANCEL pressed while in State_Battle, the BattleActionState doesn't advance, it keeps paused."
Didn't even knew about this feature, I always smash enter to get it running faster \*g\*

Reply:
Me too! But I learned from these PRs that you can just hold the key down instead of mashing it.